src/pages/AI/components/Chat/DocumentHandling.jsx

```
@@ -229,8 +225,8 @@
         formData.append("files", file);
       }
 
-      const sessionId = "unique_session_identifier_12345";
-      const apiUrl = `api/api/citation/feed-documents/?session_id=${sessionId}`;
+      const sessionId = sessionId;
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)